mmv1/third_party/terraform/services/container/node_config.go.tmpl

```diff
@@ -696,6 +829,22 @@ func schemaNodeConfig() *schema.Schema {
 								Description:      `cgroupMode specifies the cgroup mode to be used on the node.`,
 								DiffSuppressFunc: tpgresource.EmptyOrDefaultStringSuppress("CGROUP_MODE_UNSPECIFIED"),
 							},
+							"transparent_hugepage_enabled": {
+								Type:             schema.TypeString,
+								Optional:         true,
+								Computed:         true,
+								ValidateFunc:     validation.StringInSlice([]string{"TRANSPARENT_HUGEPAGE_ENABLED_ALWAYS", "TRANSPARENT_HUGEPAGE_ENABLED_MADVISE", "TRANSPARENT_HUGEPAGE_ENABLED_NEVER", "TRANSPARENT_HUGEPAGE_ENABLED_UNSPECIFIED"}, false),
```

Comment:
Do you know if new values are likely to be added in the service side in the future?

Reply:
It is a breaking change to remove values.

It is not a breaking change to add values, but there are drifts between Terraform provider and API side if new values are added in API side. New values are also needed to be added to the Terraform provider.